lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -284,16 +285,40 @@ spec = do
                 ]
 
         -- Quit delegation altogether.
-        quitStakePool @n ctx (src, fixturePassphrase) >>= flip verify
+        rq <- quitStakePool @n ctx (src, fixturePassphrase)
+        verify rq
+            -- pending tx for quitting pool is initially outgoing
+            -- because there is a fee for this tx
             [ expectResponseCode HTTP.status202
             , expectField (#status . #getApiT) (`shouldBe` Pending)
             , expectField (#direction . #getApiT) (`shouldBe` Outgoing)
             ]
+        let txid = getFromResponse Prelude.id rq
+        let (Quantity quitFeeAmt) = getFromResponse #amount rq
+        let finalQuitAmt = Quantity (depositAmt ctx - quitFeeAmt)
```

Comment:
Shouldn't this be the opposite? `quitFeeAmt - depositAmt` ?

Reply:
I changed depositAmt to be 1 000 000 lovelace (to satisfy minUTxO value), so it is bigger than the quit fee now...

Reply:
Okay, my bad. That's **quitting**, so we are getting the deposit back. The API returns `deposit - fee`, the expression above translates to: `deposit - (deposit - fee)` = `fee`. OK.